tests/features/social-feed.feature

```diff
@@ -17,7 +17,7 @@ Feature: Social feed paragraph.
     And I click "Add content"
     And I fill in "Title" with "Social feed paragraph test"
     And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
-    And I press "Add Social feed"
+    And I press "Add Social feed - Deprecated"
```

Comment:
1. If this paragraph type is no longer supported, why are we still testing it?
2. We put validation in oe_media that prevents you creating this media Shouldn't we do the same here? Put a validation that if you create a paragraph of this type, it won't allow you to do so. Or better yet, did you try removing this paragraph type from the list of selectable paragraphs at the field level? After a quick try, it seems to do the trick. The next time the user edits the node and tries to save it, they are informed they cannot reference that paragraph and they will have to remove it. And they won't have it in the selectable choices of paragraphs.

Reply:
1. Because we usually first mark features or entities as deprecated an remove them in a future release.
2. There is no validation in oe_media, the Webtools social feed media type is still available but marked as deprecated and new items can no longer be created because the "smk" services has been removed from oe_webtools.

Reply:
@22Alexandra 1. yes, but this is not working in webtools anymore. So why spend resources in testing it if anyway nobody can actually use it anymore?

Reply:
Right, I removed the test.